scripts/mesh_nbody_benchmark.py

```diff
@@ -1,3 +1,7 @@
+import sys
+sys.path.insert(0,'..') 
```

Comment:
```suggestion
sys.path.insert(0,'..') 
```

Reply:
perfect @EiffL , I removed this now from here and also from the pyramid script.
Originally I left them on purpose, in case we wanted to test this inside the same singularity image (since we can't pip install flowpm there).